### PR TITLE
[Metricbeat] Add collecting AuroraDB metrics in rds metricset

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?

This PR is to add collecting AuroraDB metrics into rds metricset: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Monitoring.html

## How to test this PR locally
1. create an AuroraDB service in AWS
2. enable aws module
3. change aws.yml config to:
```
- module: aws
  period: 5m
  credential_profile_name: elastic-test
  metricsets:
    - rds
```
4. You should see rds metrics from AuroraDB like:
`aws.rds.aurora_bin_log_replica_lag`, `aws.rds.aurora_replica.lag.ms`, `aurora_replica.lag_max.ms` and etc.

## Related issues
- Closes https://github.com/elastic/beats/issues/14142

### CODE REVIEW

File: x-pack/metricbeat/module/aws/rds/data.go [hunk 1]
```diff
@@ -75,5 +76,34 @@ var (
...
+		"backtrack_window.alert":  c.Float("BacktrackWindowAlert"),
+
+		"storage_used.backup_retention_period.bytes": c.Float("BackupRetentionPeriodStorageUsed"),
+		"storage_used.snapshot.bytes":                c.Float("SnapshotStorageUsed"),
+
+		"cache_hit_ratio.buffer":     c.Float("BufferCacheHitRatio"),
+		"cache_hit_ratio.result_set": c.Float("ResultSetCacheHitRatio"),
+
+		"engine_uptime.sec":                        c.Float("EngineUptime"),
+		"volume_read_io_per_sec":                   c.Float("VolumeReadIOPs"),
```
Comment (ChrsMark) [inline x-pack/metricbeat/module/aws/rds/data.go]: Maybe:

`volume.io_per_sec.read` and `volume.io_per_sec.write`?

Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/aws/rds/data.go]: How about `volume.read.iops` and `volume.write.iops`? I think` iops` is the unit of these two metrics. That's why I put it at the end of the metric name.

**Comment by kaiyan-sheng** [comment]
> Any plans for new visualisations for these new metrics?

Hmm I'm not planning to add one in this PR at least. rds metricset has it's predefined dashboard for general rds monitoring metrics(for all different DBs). I'm not sure how beneficial it will be to add a specific dashboard for AuroraDB only.

File: x-pack/metricbeat/module/aws/rds/data.go (hunk 1 above)
Comment (ChrsMark) [reply x-pack/metricbeat/module/aws/rds/data.go]: 👍

**Pull request merged**